rts/lra/lra-test/lra-test-arquillian-extension/src/main/java/io/narayana/lra/arquillian/spi/NarayanaLRARecovery.java

```diff
@@ -1,46 +1,43 @@
 package io.narayana.lra.arquillian.spi;
 
 import io.narayana.lra.LRAConstants;
-import io.narayana.lra.client.NarayanaLRAClient;
-import org.eclipse.microprofile.lra.tck.service.spi.LraRecoveryService;
+import org.eclipse.microprofile.lra.tck.service.spi.LRARecoveryService;
 
-import javax.enterprise.context.ApplicationScoped;
 import javax.ws.rs.client.Client;
 import javax.ws.rs.client.ClientBuilder;
 import javax.ws.rs.client.WebTarget;
 import javax.ws.rs.core.Response;
 import java.net.URI;
 
-@ApplicationScoped
-public class NarayanaLRARecovery implements LraRecoveryService {
+public class NarayanaLRARecovery implements LRARecoveryService {
 
     @Override
-    public void triggerRecovery() {
-        recoverLRAs(
-            System.getProperty(NarayanaLRAClient.LRA_COORDINATOR_HOST_KEY, "localhost"),
-            Integer.parseInt(System.getProperty(NarayanaLRAClient.LRA_COORDINATOR_PORT_KEY, "8080")));
+    public void waitForCallbacks(URI lraId) {
+        // no action is needed, tck callbacks calls are sufficiently fast for TCK
```

Comment:
This makes me wonder if https://github.com/eclipse/microprofile-lra/blob/master/tck/src/main/java/org/eclipse/microprofile/lra/tck/service/spi/LRARecoveryService.java#L49 could have a default?

Reply:
I believe that now with the PR #1530 we will need 1s waits (this replaces shortDelays). But of course, this can be added to the spec. Thanks for the pointer @tomjenkinson.

Reply:
The code base in this LRA branch does not currently need an implementation. 
I am okay to resolve this when I/we merge the LRA branch into master - ie we should decide then whether or not a default or an implementation is required.